src/event-handler/index.ts

```diff
@@ -14,6 +14,8 @@ export function createEventHandler(options: Options<any>) {
 
   return {
     on: on.bind(null, state),
+    onAny: on.bind(null, state).bind(null, "*"),
+    onError: on.bind(null, state).bind(null, "error"),
```

Comment:
This is probably not the best solution. But it ensures backwards compatibility and it works

Reply:
I would want to log a deprecation message when `.on()` is called with either `*` or `error`. While this is a neat trick, it won't allow us to do that.

I'd also like to add tests for `.on("error", handler)` and `.on("*", handler)` to a new `test/integration/deprecations-test.js` file, similar to what we have in other @octokit repositories. That makes it easier to prepare new breaking releases, and having these tests prevents accidental regressions. The tests should test for the deprecation waring to be logged

Reply:
btw you can also bind as many arguments as you like, so you could do this instead: 

```js
    onAny: on.bind(null, state, "*"),
    onError: on.bind(null, state, "error"),
```

Reply:
Done